login-workflow/docs/authentication-workflow.md

```diff
@@ -2,8 +2,73 @@
 
 The authentication workflow includes screens related to user authentication including Login, Forgot Password, Reset Password, etc.
 
+This guide covers two approaches for implementing login/authentication workflows: Okta and Custom Login.
 
-## AuthContextProvider
+
+## Okta Approach
+The Okta approach leverages Okta's identity and access management services to handle user authentication. This workflow includes screens related to user authentication such as Okta Redirect Login, Forgot Password, Reset Password, etc. By integrating with Okta, you can offload the complexities of authentication and focus on building your application.
+
+### OktaAuthContextProvider
+
+The screens in this workflow access shared data / configuration / API definitions through an `OktaAuthContextProvider` which should wrap all of the relevant routes / screens.
+
+You must supply the `OktaAuthContextProvider` with the following props / data:
+-   `language`: configures the language displayed on the screens
+-   `navigate`: a function that can be called to navigate to a new route
+-   `routeConfig`: an object describing the URLs you are using for the relevant routes so the workflow can correctly navigate between screens
+
+More information about the required and optional props can found in the [API](#okta-auth-context-provider) section.
+
+### Example
+
+Here is an example of how you would set up the Okta Redirect Login workflow using our recommended routing solution ([React Router](https://reactrouter.com/)).
+
+Each feature/screen from the Okta Auth Workflow that you wish to use should be rendered on a separate route.
+
+```tsx
+<Routes>
+    {/* Wrap all routes in a single shared OktaAuthContextProvider */}
+    <Route
+        element={
+            <OktaAuthContextProvider
+                language={'en'}
+                navigate={navigate}
+                routeConfig={{}}
+            >
+                <Outlet />
+            </OktaAuthContextProvider>
+        }
+    >
+        {/* Routes for each workflow screen you want to include */}
+        <Route/>
```

Comment:
Just show the login screen here. I realized just now that the other screens (Password stuff) won't work in this ContextProvider because the actions don't exist. They'll need to wrap those in a separate context provider if they intend to use them in combination with Okta login.